ee/clickhouse/views/cohort.py

```diff
@@ -35,6 +48,52 @@ def insert_cohort_people_into_pg(cohort: Cohort):
     cohort.insert_users_list_by_uuid(items=[str(id[0]) for id in ids])
 
 
+def insert_cohort_actors_into_ch(cohort: Cohort, filter_data: Dict):
+    insight_type = filter_data.get("insight")
+
+    if insight_type == INSIGHT_TRENDS:
+        filter = Filter(data=filter_data)
+        entity = get_target_entity(filter)
+        query, params = ClickhouseTrendsActors(cohort.team, entity, filter).actor_query()
+    elif insight_type == INSIGHT_STICKINESS:
+        stickiness_filter = StickinessFilter(data=filter_data, team=cohort.team)
+        entity = get_target_entity(stickiness_filter)
+        query, params = ClickhouseStickinessActors(cohort.team, entity, stickiness_filter).actor_query()
+    elif insight_type == INSIGHT_FUNNELS:
+        funnel_filter = Filter(data=filter_data)
+        if funnel_filter.correlation_person_entity:
+            query, params = FunnelCorrelationActors(filter=funnel_filter, team=cohort.team).actor_query()
+        else:
+            funnel_actor_class = get_funnel_actor_class(funnel_filter)
+            query, params = funnel_actor_class(filter=funnel_filter, team=cohort.team).actor_query()
+    elif insight_type == INSIGHT_PATHS:
+        path_filter = PathFilter(data=filter_data)
```

Comment:
This runs into the same issue as on caching, where if path has an accompanying `funnel_filter`, we're borked.

I think we should solve it like how Karl said: https://github.com/PostHog/posthog/issues/6658#issuecomment-952717419 - whenever we get to it.

And remove the `funnel_filter` parameter from `ClickhousePathsActors`

Reply:
Should we solve this as part of this PR?

Reply:
I'd suggest just doing it separately. This PR has been out for a while and that code path isn't used yet. We should fix the impacted areas in one go as a follow up

Reply:
I'd suggest just doing it separately. This PR has been out for a while and that code path isn't used yet. We should fix the impacted areas in one go as a follow up